editor/editor_properties_array_dict.cpp

```diff
@@ -480,6 +482,26 @@ bool EditorPropertyArray::_is_drop_valid(const Dictionary &p_drag_data) const {
 		return true;
 	}
 
+	if (drop_type == "nodes") {
+		Array nodes = drag_data["nodes"];
+
+		if (allowed_type == "NodePath") {
+			return true;
+		}
+
+		bool is_drop_allowed = true;
+		for (int i = 0; i < nodes.size(); i++) {
+			Node *dropped_node = get_tree()->get_edited_scene_root()->get_node_or_null(nodes[i]);
+			ERR_FAIL_NULL_V(dropped_node, false);
```

Comment:
I agree here you need the node_or_null variant to catch the error but maybe you should add a text to the error you put just after so that the error is not just Condition is false which imo would be just as bad as the `get_node` failing directly.

on another node why do you need the `get_tree()->get_edited_scene_root()` since `nodes[i]` is an absolute path anyway it will be called on the root anyway. This already what you do when actually dropping them so I think you should do the same here

Reply:
What should be the message ? Would `Could not get the dropped node by its node path.` be sufficient ? I